From 949abc726eab7477cee095d7696eca57247c8572 Mon Sep 17 00:00:00 2001
From: Marcel Ziswiler <marcel.ziswiler@toradex.com>
Date: Sun, 24 Mar 2019 03:31:04 +0100
Subject: [PATCH 04/19] apalis_imx6: add device tree to makefile

Add device tree to Makefile to avoid newly introduced error:

Device Tree Source is not correctly specified.
Please define 'CONFIG_DEFAULT_DEVICE_TREE'
or build with 'DEVICE_TREE=<device_tree>' argument

make[1]: *** [dts/Makefile:28: arch/arm/dts/imx6-apalis.dtb] Error 1
make: *** [Makefile:1009: dts/dt.dtb] Error 2

Signed-off-by: Marcel Ziswiler <marcel.ziswiler@toradex.com>
Reviewed-by: Igor Opaniuk <igor.opaniuk@toradex.com>

Series-to: u-boot@lists.denx.de
Series-cc: Stefano Babic <sbabic@denx.de>

Series-version: 3

Series-changes: 2
- Drop adding the colibri-imx6 device tree as Stefano already pulled v1
  75992d0e7dfc ("apalis/colibri_imx6: add device trees to makefile") but
  somehow dropped adding the apalis-imx6 one as well.

Series-changes: 3
- Added Igor's reviewed-by tag.
---
 arch/arm/dts/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index dfa5b02958..e56a39e0b1 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -561,6 +561,7 @@ dtb-$(CONFIG_MX6ULL) += \
 	imx6ull-dart-6ul.dtb
 
 dtb-$(CONFIG_ARCH_MX6) += \
+	imx6-apalis.dtb \
 	imx6-colibri.dtb
 
 dtb-$(CONFIG_MX7) += imx7d-sdb.dtb \
-- 
2.14.5

